plugins/obs-ffmpeg/obs-ffmpeg-srt.h

```diff
@@ -500,7 +503,24 @@ static int libsrt_setup(URLContext *h, const char *uri)
 	}
 
 	cur_ai = ai;
-
+	if (s->mode == SRT_MODE_RENDEZVOUS) {
+		if (s->localip == NULL || s->localport == NULL) {
+			blog(LOG_ERROR, "Invalid adapter configuration\n");
+			return OBS_OUTPUT_CONNECT_FAILED;
+		}
+		blog(LOG_DEBUG,
+		     "[obs-ffmpeg mpegts muxer / libsrt]: Adapter options %s:%s\n",
+		     s->localip, s->localport);
+		int lp = strtol(s->localport, NULL, 10);
+		if (lp <= 0 || lp >= 65536) {
+			blog(LOG_ERROR,
+			     "[obs-ffmpeg mpegts muxer / libsrt]: Local port missing in uri\n");
```

Comment:
```suggestion
			     "[obs-ffmpeg mpegts muxer / libsrt]: Local port missing in URI\n");
```
Or URL if this is, in fact, a URL.

Reply:
fixed